syntax = "proto3";
option cc_enable_arenas = true;

package Ydb;
option java_package = "com.yandex.ydb.common";
option java_outer_classname = "CommonProtos";

message FeatureFlag {
    enum Status {
        STATUS_UNSPECIFIED = 0;
        ENABLED = 1;
        DISABLED = 2;
    }
}

message CostInfo {
    // Total amount of request units (RU), consumed by the operation.
    double consumed_units = 1;
}

message QuotaExceeded {
    bool disk = 1;
}

// Specifies a point in database time
message VirtualTimestamp {
    uint64 plan_step = 1;
    uint64 tx_id = 2;
}
